en-US/Conductors_Companion/03_RPG_Preparation_Process/Session_Zero/Safety_Tools_Sensitive_Topics.md

```diff
@@ -1,4 +1,22 @@
 #### Potentially Sensitive Topics
 
-> **Warning**
-> This section has not yet been transferred from the Google Doc.
+When you discuss safety tools, describe potentially sensitive topics that might come up in an adventure or campaign.
+The following list presents a number of topics that are good to talk about, but this is not an exhaustive list.
+
+|                   |                   |                     |                |
+|:------------------|:------------------|:--------------------|:---------------|
+| Blood             | Genocide          | Murder              | Sexual assault |
+| Body horror       | Gore              | Paralysis           | Sexual contact |
+| Burning           | Harm to animals   | Physical restraint  | Slavery        |
+| Cancer            | Harm to children  | Racism              | Spiders        |
+| Cannibalism       | Homophobia        | Rats                | Starvation     |
+| Claustrophobia    | Incest            | Real-world religion | Terrorism      |
+| Death by exposure | Insects           | Ritual sacrifice    | Thirst         |
+| Freezing          | Kidnapping        | Self-harm           | Torture        |
+| Gaslighting       | Mental domination | Sexism              | Transphobia    |
```

Comment:
This was a list, but I made it into a table to reduce it's length without loosing information.

<img width="495" height="409" alt="The rendered version of this file" src="https://github.com/user-attachments/assets/5478e8d4-cc71-4eb7-8d6c-bfe44398e5d3" />

It's not perfect with that empty header row, but it's 1/4 the length of a list with the same contents.